ansible/tasks/deploy_strapi_test_prod.yaml

```diff
@@ -0,0 +1,17 @@
+---  
+  - name: deploy {{ react_infra_name }} of PR {{ PR }} in {{ namespace }} namespace
```

Comment:
should this be 'react_infra_name' or is there a 'strapi_infra_name'

Reply:
Good catch.